Alterações requisitadas

lcasviana opened the pull request:
:D

LindleyWerner committed:
Criação de mais um minijogo

LindleyWerner committed:
Retirada de divergencias para o pull request

lcasviana committed:
Novas alterações...

fegemo commented:
@lcasviana, ainda não executei, mas olhando para o código parece que vocês optaram por não corrigir a gravidade do pulo, certo? Me parece também que o erro da música de fundo continuar tocando ainda permanece no MiniGame. Isso procede?

lcasviana committed:
Corrigido bug do som.

lcasviana committed:
Bug som
Merge branch 'master' of https://github.com/fegemo/cefet-games-brushmyteethplz

lcasviana committed:
......
Merge branch 'master' of https://github.com/lcasviana/cefet-games-brushmyteethplz

lcasviana committed:
123123

lcasviana committed:
asdasd
Merge branch 'master' of https://github.com/lcasviana/cefet-games-brushmyteethplz

lcasviana commented:
Corrigido bug do som.
Eu nao sei se o pull request ta aberto e vai atualizando porque eu nao consegui requisitar um novo.

fegemo commented on core/src/br/cefetmg/games/minigames/SideWalking.java:
```diff
@@ -8,6 +8,10 @@
 import br.cefetmg.games.graphics.MultiAnimatedSprite;
 import br.cefetmg.games.minigames.util.DifficultyCurve;
 import br.cefetmg.games.minigames.util.GameStateObserver;
+<<<<<<< HEAD
```
Enviou conflitos sem resolução.

fegemo reviewed (changes requested):
Resolva os conflitos antes de enviar. O código enviado está completamente quebrado (não compila) por causa dos conflitos não resolvidos.
